src/dataExplorer/context/tags.tsx

```diff
@@ -158,6 +166,41 @@ export const TagsProvider: FC<Prop> = ({children, scope}) => {
       [tagKey]: RemoteDataState.Loading,
     })
 
+    if (isFlagEnabled('v2privateQueryUI') && isIOx) {
+      const queryTextSQL: string = `
+        SELECT DISTINCT("${tagKey}")
+        FROM "${measurement}"
+        ORDER BY "${tagKey}"
+        LIMIT ${DEFAULT_LIMIT}
+      `
+      try {
+        const resp = await queryAPI(queryTextSQL, scope, {
+          language: LanguageType.SQL, // use SQL to get measurement list
+          bucket,
+        } as QueryOptions)
```

Comment:
I just realize that I have been using `as` in an incorrect way 😅 I was intended to show the type of an object, and I thought `as` is a way to do so. Apparently it is not a good habit since `as` usually implies there is something wrong with the type in the code. Sorry for the confusing! Updated https://github.com/influxdata/ui/pull/6785/commits/c3b477dabbfb25edfcfd0f303f22848d9337e4c8